src/Tests/Microsoft.NET.ToolPack.Tests/GivenThatWeWantToPackAToolProjectWithExplicitConfig.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+using NuGet.Packaging;
+
+namespace Microsoft.NET.ToolPack.Tests
+{
+    public class GivenThatWeWantToPackAToolProjectWithExplicitConfig : SdkTest
+    {
+
+        public GivenThatWeWantToPackAToolProjectWithExplicitConfig(ITestOutputHelper log) : base(log)
+        {
+
+        }
+
+        [Fact]
+        public void It_finds_the_entry_point_dll_and_put_in_setting_file()
+        {
+            const string explicitEntryPoint = "explicit_entry_point.dll";
+            TestAsset helloWorldAsset = _testAssetsManager
+            .CopyTestAsset("PortableTool", "PackPortableToolToolEntryPoint")
```

Comment:
Nit: I think we generally indent the subsequent lines of a chained multi-line statement like this.

Reply:
Fixed